storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc

```diff
@@ -20,20 +20,20 @@ Familiarity with link:https://access.redhat.com/documentation/en-us/openshift_co
 
 After installing the AWS EFS CSI Driver Operator, {product-title} installs the AWS EFS CSI Operator and the AWS EFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the AWS EFS CSI Driver Operator to create CSI-provisioned PVs that mount to AWS EFS assets.
```

Comment:
@smunje1 don't we need to update the reference 'AWS EFS CSI driver' to 'Amazon EFS CSI driver'?

Reply:
I agree that this text should be updated to "Amazon EBS".

Reply:
We should be careful in the situations where we're referring to two very similar things....
https://github.com/openshift/aws-ebs-csi-driver-operator  is the Red Hat authored operator that installs and maintains the https://github.com/openshift/aws-ebs-csi-driver (a fork of https://github.com/kubernetes-sigs/aws-ebs-csi-driver).
The aws-ebs-csi-driver is the name of the CSI driver, and its name should not be altered or expanded in documentation. We should not alter our repo ame references or operator name references. to match nomenclature external to GitHub repo names.

Specific to EFS, we have a similar situation with https://github.com/openshift/aws-efs-csi-driver-operator that installs and maintains for EFS. The fork is https://github.com/openshift/aws-efs-csi-driver from https://github.com/kubernetes-sigs/aws-efs-csi-driver.

As a result of the above, I disagree that we should change the documentation reference of 'AWS EFS CSI driver' to 'Amazon EFS CSI driver'.

@codymant please ack.
@smunje1 please ack.

Reply:
Ack.